ci-operator/step-registry/openshift/e2e/gcp/cilium/openshift-e2e-gcp-cilium-workflow.yaml

```diff
@@ -28,6 +28,7 @@ workflow:
         Pods cannot access the /config/master API endpoint\|
         Services should be able to update service type to NodePort listening on same port number but different protocols\|
         Services should respect internalTrafficPolicy=Local Pod to Pod\|
+        Services should respect internalTrafficPolicy=Local Pod (hostNetwork: true) to Pod\|
```

Comment:
Needs to have comment updated above with explanation

Reply:
numbering seems to be wrong before, so it doesn't need any change as this test also belongs to `pending investigation from Cilium team`